types/TestResult.js

```diff
@@ -55,6 +83,12 @@ export type AssertionResult = {|
   title: string,
 |};
 
+export type FormattedTestAssertion = {
```

Comment:
which type does this correspond to? AssertionResult? If so, shall we call it FormattedAssertionResult?